erts/emulator/test/bif_SUITE.erl

```diff
@@ -1792,35 +1788,42 @@ busy_wait_go() ->
 
 id(I) -> I.
 
-%% Get code path, including the path for the erts application.
-get_code_path() ->
-    Erts = filename:join([code:root_dir(),"erts","preloaded","ebin"]),
-    case filelib:is_dir(Erts) of
-	true->
-	    [Erts|code:get_path()];
-	_ ->
-	    code:get_path()
+bif_modules(MFAs) ->
+    BifRel = sofs:relation(MFAs, [{m,f,a}]),
+    BifModules = sofs:to_external(sofs:projection(1, BifRel)),
+    case is_otp_installed() of
+        true ->
+            BifModules;
+        false ->
+            %% Running in an uninstalled system. The preloaded modules
+            %% don't contain any abstract code.
+            io:put_chars("** Running in an uninstalled system. Not verifying pre-loaded modules\n"
+                         "** because they don't have debug_info chunks.\n"),
+            [M || M <- BifModules, code:which(M) =/= preloaded]
     end.
 
-which(Mod, Path) ->
-    which_1(atom_to_list(Mod) ++ ".beam", Path).
+is_otp_installed() ->
+    LibDir = code:lib_dir(),
+    ErtsDir = code:lib_dir(erts),
+    lists:prefix(LibDir, ErtsDir).
 
-which_1(Base, [D|Ds]) ->
-    Path = filename:join(D, Base),
-    case filelib:is_regular(Path) of
-	true -> Path;
-	false -> which_1(Base, Ds)
-    end.
 print_mfa({M,F,A}) ->
     io:format("~p:~p/~p", [M,F,A]).
 
-extract_abstract(Mod, Path) ->
-    Beam = which(Mod, Path),
+which(Mod) ->
```

Comment:
I think that this should work in an uninstalled system as well. Have you tried?

Reply:
No...

... so there is an `ebin` directory directly under the `erts` directory.

I've forced-pushed a simplified version.